oneandone/src/test/java/org/apache/jclouds/oneandone/rest/internal/BaseOneAndOneApiMockTest.java

```diff
@@ -107,4 +111,11 @@ protected RecordedRequest assertSent(MockWebServer server, String method, String
       assertEquals(parser.parse(new String(request.getBody(), Charsets.UTF_8)), parser.parse(json));
       return request;
    }
+
+   protected String FormatDateToISO8601(Date date) {
+      TimeZone tz = TimeZone.getTimeZone("UTC");
+      DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+      df.setTimeZone(tz);
+      return df.format(date).concat("Z");
+   }
```

Comment:
Same as before, better use the DateService. You can get it from the Guice injector.